experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/PopulationSizing/index.tsx

```diff
@@ -0,0 +1,286 @@
+
+const PopulationSizing = ({
+  sizingData,
+  totalNewClients,
+  totalExistingClients,
+}: PopulationSizingProps) => {
+  const newUserSizing = sizingData.new;
+  const existingUserSizing = sizingData.existing;
+  const targetingParameters = JSON.stringify(
+    newUserSizing.target_recipe,
+    (k, v) => (k === "new_or_existing" ? undefined : v),
+  );
+  return (
+    <>
+      <Form.Label
+        className="d-flex align-items-center"
+        data-testid="population-sizing-precomputed-values"
+      >
+        Pre-computed population sizing data
+        <Info
+          data-tip
+          data-for="auto-sizing-help"
+          width="20"
+          height="20"
+          className="ml-1"
+        />
+        <TooltipWithMarkdown
+          tooltipId="auto-sizing-help"
+          markdown={popSizingHelpMarkdown}
+        />
+      </Form.Label>
+      <div>
+        <p className="text-secondary">
+          This is meant as a guide for certain targeting parameters. As always,
+          contact Data Science in{" "}
+          <LinkExternal href="https://mozilla.slack.com/archives/CF94YGE03">
+            #ask-experimenter
+          </LinkExternal>{" "}
+          for assistance.
+        </p>
+      </div>
+      <Form.Group>
+        <Form.Label className="d-flex align-items-center">
+          Targeting Parameters
+        </Form.Label>
+        <Code codeString={targetingParameters} />
+      </Form.Group>
+      <Form.Label
+        className="d-flex align-items-center"
+        data-testid="new-total-clients-label"
+      >
+        {totalNewClients} total{" "}
+        <span
+          style={{ fontWeight: 700, marginLeft: "0.3em", marginRight: "0.3em" }}
+        >
+          new
+        </span>{" "}
+        clients for given parameters
+      </Form.Label>
+      <Form.Label
+        className="d-flex align-items-center"
+        data-testid="existing-total-clients-label"
+      >
+        {totalExistingClients} total{" "}
+        <span
+          style={{ fontWeight: 700, marginLeft: "0.3em", marginRight: "0.3em" }}
+        >
+          existing
+        </span>{" "}
+        clients for given parameters
+      </Form.Label>
+      <p>
+        {Object.keys(newUserSizing.sample_sizes).map((powerKey) => {
+          // powerKey is the same for new and existing so we can look both up with the same key
+          const newUserMetrics =
+            newUserSizing.sample_sizes[powerKey]["metrics"];
+          const existingUserMetrics =
+            existingUserSizing.sample_sizes[powerKey]["metrics"];
+          // parameters are the same for new and existing so we only need to get one
+          const parameters = newUserSizing.sample_sizes[powerKey]["parameters"];
+          return (
+            <>
+              <Form.Label style={{ fontWeight: 700 }}>
+                Effect Size: {parameters.effect_size * 100}%
```

Comment:
Do we need to `.toFixed` this too in case it winds up being a wonky float?

Reply:
Sure! I don't think it's necessary, but it certainly doesn't hurt.

Reply:
Hm I was planning to do `Effect Size: {parameters.effect_size.toFixed(3) * 100}%`, but `toFixed` returns a string, which typescript doesn't want me to multiply by 100. I can parse it back to a float but then we're back where we were, so I guess I'll have to do `.toFixed(1)` after the `* 100` part. Seems slightly worse aesthetically, but at least it's much better than if we ended up with something like 5.00000000000000012723% somehow.